test/Microsoft.AspNet.Server.Kestrel.FunctionalTests/ResponseTests.cs

```diff
@@ -125,6 +126,48 @@ public class ResponseTests
             }
         }
 
+        [Fact]
+        public async Task OnCompleteCalledEvenWhenOnStartingNotCalled()
+        {
+            var config = new ConfigurationBuilder()
+                .AddInMemoryCollection(new Dictionary<string, string>
+                {
+                    { "server.urls", "http://localhost:8793/" }
```

Comment:
Not sure how much this really matters but the above tests were incrementing port numbers by one, so this one would use 8794 to follow the pattern. Might as well be consistent.